database/migrations/2020_05_10_015415_create_hh5p_libraries_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateHH5pLibrariesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('hh5p_libraries', function (Blueprint $table) {
```

Comment:
`hh5p_libraries` or its typo and should be `h5p_libraries `

Reply:
first `h` (headless)